client/src/components/screens/PatientSubmission/index.jsx

```diff
@@ -66,6 +66,24 @@ const mrnValue = (patient) => {
   return '';
 };
 
+const ethnicityValue = (patient) => {
+  const { extension } = patient;
+  if (find(extension, o => o.url.includes('qc-ethnicity') && o.valueCoding.code)) {
+    return find(extension, o => o.url.includes('qc-ethnicity')).valueCoding.code;
+  }
+
+  return undefined;
+};
+
+const consanguinityValue = (patient) => {
+  const { extension } = patient;
+  if (find(extension, o => o.url.includes('blood-relationship') && o.valueCoding.code)) {
```

Comment:
You could store the value of `find(extension, o => o.url.includes('blood-relationship')` in a `const` variable, and use it again in line afterwards so you don't have to call the `find` function again. 
Also, this function and `ethnicityValue` share the same logic, maybe you could create a generalized function to be used by both? Something like `const getValueCoding = (patient, exensionName) => {...}` which would return the `valueCoding`